release.gradle

```diff
@@ -0,0 +1,181 @@
+        artifactId BASENAME
+        version VERSION
+        description DESC
+
+        pom.withXml {
+            def root = asNode()
+            root.children().last() + pomConfig
+            root.appendNode("description", DESC) //TODO Find why description is missing from pom.xml
+
+            final dependenciesNode = asNode().appendNode('dependencies')
+            ext.addDependency = { Dependency dep, String scope ->
+                if (dep.group == null || dep.version == null || dep.name == null || dep.name == "unspecified") {
+                    return // ignore invalid dependencies
+                }
+
+                // Add InstantSearch prefix to our modules when releasing
+                //TODO: Find a way to avoid listing modules manually
+                final moduleNames = ["core", "ui"]
+                final depFinalName = dep.group == "com.algolia" && moduleNames.contains(dep.name) ?
+                        "instantsearch-android-$dep.name" : dep.name
+                println "Adding dependency $depFinalName ($scope) to pom.xml"
+
+                final dependencyNode = dependenciesNode.appendNode('dependency')
+                dependencyNode.appendNode('groupId', dep.group)
+                dependencyNode.appendNode('artifactId', depFinalName)
+                dependencyNode.appendNode('version', dep.version)
+                dependencyNode.appendNode('scope', scope)
+
+                if (!dep.transitive) {
+                    // If this dependency is transitive, we should force exclude all its dependencies them from the POM
+                    final exclusionNode = dependencyNode.appendNode('exclusions').appendNode('exclusion')
+                    exclusionNode.appendNode('groupId', '*')
+                    exclusionNode.appendNode('artifactId', '*')
+                } else if (!dep.properties.excludeRules.empty) {
+                    // Otherwise add specified exclude rules
+                    final exclusionNode = dependencyNode.appendNode('exclusions').appendNode('exclusion')
+                    dep.properties.excludeRules.each { ExcludeRule rule ->
+                        exclusionNode.appendNode('groupId', rule.group ?: '*')
+                        exclusionNode.appendNode('artifactId', rule.module ?: '*')
+                    }
+                }
+            }
+
+            // List all "compile" dependencies (for old Gradle)
+            configurations.compile.getAllDependencies().each { dep -> addDependency(dep, "compile") }
+            // List all "api" dependencies (for new Gradle) as "compile" dependencies
+            configurations.api.getAllDependencies().each { dep -> addDependency(dep, "compile") }
+            // List all "implementation" dependencies (for new Gradle) as "runtime" dependencies
+            configurations.implementation.getAllDependencies().each { dep -> addDependency(dep, "runtime") }
+        }
+    }
+}
+
+group = GROUP
+version = VERSION
+
+bintray {
```

Comment:
Can't we just apply the bintray template that we have for this ?

Reply:
Yes, we should! I've been adapting here but forgot to backport to `instantsearch-mobile-tools`. 

Done now, does it LGTY?